src/encoder/plugins/OpusEncoderPlugin.cxx

```diff
@@ -84,7 +88,7 @@ class PreparedOpusEncoder final : public PreparedEncoder {
 	Encoder *Open(AudioFormat &audio_format) override;
 
 	const char *GetMimeType() const override {
-		return "audio/ogg";
+		return "audio/ogg; codecs=opus";
```

Comment:
This change has nothing to do with chaining feature, does it? Should be a separate commit; I can cherry-pick such a tiny commit easily, even if there are issues left with the "large" commit.

Reply:
it has nothing to do with chaining feature, yes

Reply:
From RFC 7845:
   An "Ogg Opus file" consists of one or more sequentially multiplexed
   segments, each containing exactly one Ogg Opus stream.  The
   RECOMMENDED mime-type for Ogg Opus files is "audio/ogg".

   If more specificity is desired, one MAY indicate the presence of Opus
   streams using the codecs parameter defined in [RFC6381] and
   [RFC5334], e.g.,

                            audio/ogg; codecs=opus

   for an Ogg Opus file.

so I guess I should leave it as audio/ogg, as it's recommended